### PR TITLE
kubenetes tasks

### PR DESCRIPTION
**Pull request by Mark-Powers**
Adds tasks to 
- label nodes automatically as devices join kubernetes
- cleanup namespaces once all pods are finished

### CODE REVIEW

File: floto/api/kubernetes.py [hunk 1]
```diff
@@ -33,11 +33,26 @@ def get_volume_name():
...
+    core_api.patch_node(
+        node_name,
+        {
+            "metadata": {
+                "labels": {
+                    "node-role.kubernetes.io/floto-worker": "true",
+                }
+            }
+        }
+    )
```
Comment (msherman64) [inline floto/api/kubernetes.py]: commenting our in-person convo:

this seems to use the same value for the label_selector as for the payload, so we should use something else for the selector?

File: floto/settings.py [hunk 2]
```diff
@@ -302,8 +303,12 @@
...
+    "label_nodes": {
+        "task": "label_nodes",
+        "schedule": crontab(minute="*/5"),
+    },
     "cleanup_namespaces": {
         "task": "cleanup_namespaces",
-        "schedule": crontab(minute="0"), # every hour
-    }
+        "schedule": crontab(minute="*/30"),
+    },
```
Comment (msherman64) [inline floto/settings.py:313]: we possibly want some jitter on this just in case something else syncs up with this task by accident.

**Review by msherman64 — APPROVED** [review]
LGTM other than the selector change

File: floto/settings.py (hunk 2 above)
Comment (Mark-Powers) [reply floto/settings.py:313]: I agree, I'll keep this in mind if we add any more tasks.

**Pull request merged**